packages/core-webhooks/src/conditions.ts

```diff
@@ -1,12 +1,16 @@
-export const between = (actual, expected): boolean => actual > expected.min && actual < expected.max;
+import { Utils } from "@arkecosystem/crypto";
+
+const toBoolean = (value): boolean => value.toString().toLowerCase().trim() === "true" ? true : false;
+
+export const between = (actual, expected): boolean => gt(actual, expected.min) && lt(actual, expected.max);
 export const contains = (actual, expected): boolean => actual.includes(expected);
 export const eq = (actual, expected): boolean => actual === expected;
-export const falsy = (actual): boolean => actual === false;
-export const gt = (actual, expected): boolean => actual > expected;
-export const gte = (actual, expected): boolean => actual >= expected;
-export const lt = (actual, expected): boolean => actual < expected;
-export const lte = (actual, expected): boolean => actual <= expected;
+export const falsy = (actual): boolean => actual === false || !toBoolean(actual);
+export const gt = (actual, expected): boolean => Utils.BigNumber.make(actual).gt(expected);
+export const gte = (actual, expected): boolean => Utils.BigNumber.make(actual).gte(expected);
+export const lt = (actual, expected): boolean => Utils.BigNumber.make(actual).lt(expected);
+export const lte = (actual, expected): boolean => Utils.BigNumber.make(actual).lte(expected);
 export const ne = (actual, expected): boolean => actual !== expected;
```

Comment:
I think we should check here if the given values are potentially in the bignumber range and then compare them as such.

Reply:
BigNumber returns `false` if any of the compared values are null or NaN so this should be fine as is.